### PR TITLE
Cadl website basic scaffolding in 11Ty

### PR DESCRIPTION
**Pull request by dtuyishime**
- [x]  Simple front page with basic file structure layout (CSS, navigation), no content is needed
image
image
image

### CODE REVIEW

File: packages/compiler/package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "mkdirp": "~1.0.4",
```
Comment (timotheeguerin) [inline packages/compiler/package.json:66]: Add a comment here that the production playground is broken and needs to be resolved before merging.

Comment (dtuyishime) [reply packages/compiler/package.json:66]: I guess this is resolved.

File: common/changes/@cadl-lang/compiler/cadl-website_2022-08-16-21-36.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (timotheeguerin) [inline common/changes/@cadl-lang/compiler/cadl-website_2022-08-16-21-36.json:1]: remove this file, not needed anymore as the change was made outside of this PR

File: rush.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
    * Options that are only used when the PNPM package manager is selected
    */
   "pnpmOptions": {
-    "strictPeerDependencies": true
+    "strictPeerDependencies": false
```
[the remark this answers is not in the document]
Comment (timotheeguerin) [reply rush.json:14]: this is not resolved

**Comment by markcowl** [comment]
- [ ] need ci pipeline for publishing

File: packages/website/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Prerequisites
+
+#### Install Node & NPM
+
+Downlaod and install from https://nodejs.org.
+
+#### Install Eleventy (11Ty) using NPM.
+
+```bash
+> npm i --save-dev @11ty/eleventy
```
Comment (timotheeguerin) [inline packages/website/README.md:18]: Don't need that step, was just needed for you the first time but now it is part of the pacakge.json

File: packages/website/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+Support libraries that enable the demo features are in the `packages/` directory. The demo code itself is in the `website/` directory.
+
+## Instructions for editing or contributing to this website
+
+### Prerequisites
+
+#### Install Node & NPM
+
+Downlaod and install from https://nodejs.org.
```
Comment (timotheeguerin) [inline packages/website/README.md:13]: Would skip this part as well as well as all the repo setup part as this is already part of the `CONTRIBUTING.MD` guide

File: packages/website/README.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+```bash
+> rush install
+```
+
+Next, build the project.
+
+```bash
+> rush build
+```
```
Comment (timotheeguerin) [inline packages/website/README.md:33]: Meant this entire section as well, this is all part of the contributing guide.

File: packages/website/README.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+### Running the website
```
Comment (timotheeguerin) [inline packages/website/README.md:35]: I would also add a step here on how to build it for production `npm run build` saying the output will be in the `_website` folder

**Comment by timotheeguerin** [comment]
Taken as part of  by https://github.com/microsoft/cadl/pull/935

**Pull request closed without merging**